routes/login.ts

```diff
@@ -99,24 +93,16 @@ router.post("/googleOauth", async (req:any, res:any, next) => {
   }
 });
 
-router.post("/forgot", async (req:any, res:any) => {
-  const usernameOrEmail = req.body.usernameEmail;
-  let regex = new RegExp("[a-z0-9]+@[a-z]+.[a-z]{2,3}");
-  let user;
-  const isInputEmail = regex.test(usernameOrEmail);
-
-  if (!isInputEmail) {
-    user = await getProfileByUsername(usernameOrEmail);
-  } else {
-    user = await getProfileByEmail(usernameOrEmail);
-  }
+router.post("/forgot", async (req: any, res: any) => {
+  const email = req.body.email;
+  const user = await getProfileByEmail(email);
 
   if (user !== null && user && user.registeredWith !== "google") {
     const resetToken = Math.floor(100000000 + Math.random() * 900000000);
-    await updateUserResetToken(Number(user.id), resetToken.toString());
+    await updateUserResetToken(user.id, resetToken.toString());
     const msg = {
       to: `${user.email}`,
-      from: "crowdwrap@gmail.com",
+      from: "lwb@gmail.com",
```

Comment:
would need to update the api for this, and also create an email with that name

Reply:
which api ?